packages/inertia-react/src/Link.js

```diff
@@ -65,5 +65,9 @@ export default forwardRef(function InertiaLink({
     ],
   )
 
+  const [url, _data] = mergeQueryStringsWithData(method, hrefToUrl(href), data)
+  href = url.href
+  data = _data
```

Comment:
The `_` is a bit icky looking, but I can't immediately come up with a better solution.

Reply:
Yup, totally agree. This even took me a while to come up with.